src/main/kotlin/parts/wisdom/arcraider/Arc.kt

```diff
@@ -1,31 +1,34 @@
 package parts.wisdom.arcraider
 
 import java.io.File
+import kotlinx.serialization.*
+import kotlinx.serialization.json.*
 
 val arcData = File(
     checkNotNull(System.getenv("ARC_DATA"),
         { "ARC_DATA environment variable is not set" })
 )
 
+
+@Serializable
 data class Task(
-    val trainingPairs: List<ArcPair>,
-    val testPairs: List<ArcPair>
+    val train: List<ArcPair>,
+    val test: List<ArcPair>
 )
 
+@Serializable
 data class ArcPair(
-    val input: ArcGrid,
-    val output: ArcGrid
+    val input: List<List<Integer>>,
+    val output: List<List<Integer>>
 )
 
-class ArcGrid() {
-}
-
 fun loadTasksFromDir(dir: File): List<Task> {
     TODO()
 }
 
 fun loadTaskFromFile(file: File): Task {
-    TODO()
+    val json = Json(JsonConfiguration.Stable)
+     return json.parse(Task.serializer(), file.readText())
```

Comment:
We can probably inline this. We don't gain much in readability by creating the variable.

Reply:
fixed